products/knownprojects/bash/04_export.sh

```diff
@@ -30,9 +30,9 @@ mkdir -p output
         csv_export combined &
         csv_export review_project &
         csv_export review_dob &
+        csv_export review_no_geometry &
         csv_export corrections_applied &
         csv_export corrections_not_applied &
-        # csv_export corrections_zap &
```

Comment:
why no corrections_zap

Reply:
it was deprecated in favor of `zap_record_ids.csv` [here](https://github.com/NYCPlanning/data-engineering/blob/kpdb-build-for-research/products/knownprojects/sql/dcp_application.sql#L108-L112)